lib/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/HistoryEventServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package pro.taskana.simplehistory.impl;
+
+import java.sql.SQLException;
+import java.time.Instant;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import pro.taskana.configuration.TaskanaEngineConfiguration;
+import pro.taskana.history.api.TaskanaHistory;
+import pro.taskana.history.api.TaskanaHistoryEvent;
+import pro.taskana.impl.util.LoggerUtils;
+import pro.taskana.simplehistory.impl.mappings.HistoryEventMapper;
+
+/**
+ * This is the implementation of HistoryService.
+ */
+public class HistoryEventServiceImpl implements TaskanaHistory {
```

Comment:
Why did you rename it to HistoryEVENTService? Is there a reason for it?

Reply:
I Thought that HistoryEventService is a name closer to the funtionality that we are going to implement.